tests/model/model.test.ts

```diff
@@ -366,4 +369,22 @@ describe("Model", () => {
       ],
     });
   });
+
+  test("snapshot when importing xlsx file", async () => {
+    jest.spyOn(console, "warn").mockImplementation(() => {});
```

Comment:
Is there a reason ?

Reply:
there is a global handler to make the tests fail in case of console.error or console.warn

Reply:
I know this global handler, but if I comment this line, the test does not fail. Expected ?

Reply:
That's weird, it was failing before 🤷